internal/kms/repository_audit_key.go

```diff
@@ -44,7 +44,10 @@ func createAuditKeyTx(ctx context.Context, r db.Reader, w db.Writer, rkvWrapper
 	if len(key) == 0 {
 		return nil, nil, errors.New(ctx, errors.InvalidParameter, op, "missing key")
 	}
-	rootKeyVersionId := rkvWrapper.KeyID()
+	rootKeyVersionId, err := rkvWrapper.KeyId(ctx)
```

Comment:
Reviewers: this file could be looked at if desired, very minor changes but repeated throughout the various key types.